### ISSUE TITLE
Faithfulness metric on imags explanations

### ISSUE THREAD

**Issue by dsvrsec** [Trusted-AI/AIX360#99]
Can the faithfulness metric can be used as a metric for explanations obtained using LIME or SHAP.

**Comment by sadhamanus** [comment]
Yes. It can used for any feature based local explainability method.

**Comment by Nadiatriki** [comment]
@sadhamanus what dose it means when faithfulness_metric  gave me 'nan' while using SHAP and LIME for explaining a Tabular data ? 
and thanks

**Comment by sadhamanus** [comment]
Hard to say exactly but the metric just computes correlation so either the feature importance are all the same or the prediction of your model hasn't changed when setting each feature to base value. In either of these cases it means all features are equally unimportant. Another possibility could be model outputting nans for different inputs it tries by setting each feature individually to base values.
